lib/grandpa/reputation.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package grandpa
+
+// This file is where the reputation change types are declared. Ones that require
+// no calculation are defined as variables, while the ones that do (or take params)
```

Comment:
maybe define all of them as functions to prevent suprise mutations? If you have a function `PastRejection()` always returning a new `peerset.ReputationChange`, there cannot be surprise mutation with the global variable